.buildkite/data.rayci.yml

```diff
@@ -29,28 +29,28 @@ steps:
 
   # tests
   - label: ":database: data: arrow v9 tests"
-    tags: 
+    tags:
```

Comment:
linter makes all these changes, otherwise I cannot push

Reply:
could you split the unrelated lint changes to another PR?

Reply:
That's a good suggestion and I thought about it also, my confusion is, when I tried to push on the master branch, prepush hook doesn't seem to work :(

Reply:
you can just checkout the changes of these files.

I think the hook is configured to fix changed files only.

the people who added the pre-commit's should have fixed all the lint errors across the entire repo first.. and then run it on CI with `-a` flag..